Bio/KEGG/Gene/__init__.py

```diff
@@ -0,0 +1,153 @@
+# Copyright 2017 by Kozo Nishida.  All rights reserved.
+# This code is part of the Biopython distribution and governed by its
+# license.  Please see the LICENSE file that should have been included
+# as part of this package.
+
+"""Code to work with the KEGG Gene database.
+
+Functions:
+parse - Returns an iterator giving Record objects.
+
+Classes:
+Record - A representation of a KEGG Gene.
+"""
+
+# other Biopython stuff
+from __future__ import print_function
+
+from Bio.KEGG import _write_kegg
+from Bio.KEGG import _wrap_kegg
+
+# Set up line wrapping rules (see Bio.KEGG._wrap_kegg)
+name_wrap = [0, "",
+             (" ", "$", 1, 1),
+             ("-", "$", 1, 1)]
+id_wrap = lambda indent: [indent, "", (" ", "", 1, 0)]
+
+
+class Record(object):
+    """Holds info from a KEGG Gene record.
+
+    Members:
+    entry       The entry identifier.
```

Comment:
Can you make this into a reStructuredText bullet point list please?

```python
class Record(object):
    """Holds info from a KEGG Gene record.

    Attributes:
     - entry       The entry identifier.
     - name        A list of the gene names.
     - definition  The definition for the gene.
     - orthology   A list of 2-tuples: (orthology id, role)
     - organism    A tuple: (organism id, organism)
     - position    The position for the gene
     - motif       A list of 2-tuples: (database, list of link ids)
     - dblinks     A list of 2-tuples: (database, list of link ids)

    """
```

e.g. https://github.com/biopython/biopython/blob/biopython-170/Bio/KEGG/__init__.py#L25

I have found http://rst.ninjs.org/ useful here. Cross reference #1221